linux/gatt/client.go

```diff
@@ -212,10 +212,30 @@ func (p *Client) ReadCharacteristic(c *ble.Characteristic) ([]byte, error) {
 }
 
 // ReadLongCharacteristic reads a characteristic value which is longer than the MTU. [Vol 3, Part G, 4.8.3]
-func (p *Client) ReadLongCharacteristic(c *ble.Characteristic) ([]byte, error) {
+func (p *Client) ReadLongCharacteristic(c *ble.Characteristic) ([]byte,error) {
 	p.Lock()
 	defer p.Unlock()
-	return nil, nil
+
+	buffer := make([]byte,512)
```

Comment:
That makes sense. In the next commit I set the initial length to 0 and capacity to 512 (max bytes that can be stored by an attribute).